src/components/skeletons/TextLineSkeleton.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from "react";
+
+export interface TextLineSkeletonProps {
+  width: number;
+}
+const TextLineSkeleton: React.FC<TextLineSkeletonProps> = ({ width }) => (
+  <div className="ssc-line w-100" style={{ width: `${width}px` }} />
```

Comment:
Is the `w-100` a class for setting width?
Isn't that a bit redundant if the width prop is required to be set?

Reply:
Good thinking!